scripts/protoc.sh

```diff
@@ -9,44 +9,47 @@
 set -e
 
 REPO=github.com/tinkerbell/pbnj
-REPO_BASE=$(dirname ${REPO})
 PROTOS_LOC=api/v1
-OUT_LOC=api/v1
+PROTOC_VERSION=3.13.0
 
 function installDeps {
     if ! which protoc 2>&1 >/dev/null; then
         echo 'Installing protoc...' >&2
-        if which apt 2>&1 >/dev/null; then
-            apt update; apt install -y protobuf-compiler
-        elif which yum 2>&1 >/dev/null; then
-            yum -y install protobuf-compiler
-        else
-            echo 'Unknown package manager' >&2
-            exit 1
+        if ! which unzip 2>&1 >/dev/null; then
+            if which apt 2>&1 >/dev/null; then
+                apt update; apt install -y zip
+            elif which yum 2>&1 >/dev/null; then
+                yum -y install zip
+            else
+                echo 'Unknown package manager' >&2
+                exit 1
+            fi
         fi
+        PB_REL="https://github.com/protocolbuffers/protobuf/releases"
+        curl -LO $PB_REL/download/v${PROTOC_VERSION}/protoc-${PROTOC_VERSION}-linux-x86_64.zip
+        unzip protoc-${PROTOC_VERSION}-linux-x86_64.zip -d /usr/local
         echo 'Protoc installed' >&2
     else
         echo 'Protoc already installed!' >&2
     fi
     go get google.golang.org/grpc/cmd/protoc-gen-go-grpc@v1.0.1
     go get google.golang.org/grpc@v1.32.0
     go get google.golang.org/protobuf/cmd/protoc-gen-go@v1.25.0
+    go get github.com/mwitkow/go-proto-validators/protoc-gen-govalidators@v0.3.2
     go get golang.org/x/tools/cmd/goimports
-    go mod tidy
+    go mod tidy || true
```

Comment:
I would think we would want it to fail if the tidy fails no? Probably means something is wrong.

Reply:
the tidy is only needed when running this script locally. when running it in the docker image build as the protobufs dependencies there's no mod file so this line actually fails. hence the need for the`|| true`.